skyplane/api/transfer_job.py

```diff
@@ -56,7 +57,9 @@ def __init__(self, src_obj: ObjectStoreObject, dst_objs: Dict[str, ObjectStoreOb
 class GatewayMessage:
     def __init__(self, chunk: Optional[Chunk] = None, upload_id_mapping: Optional[Dict[str, Tuple[str, str]]] = None):
         self.chunk = chunk
-        self.upload_id_mapping = upload_id_mapping
+        self.upload_id_mapping = (
+            upload_id_mapping  # region_tag: (upload_id, dst_key) # TODO: map both bucket + region to enable multiple buckets per region
+        )
```

Comment:
could you clarify this mapping change?

Reply:
We eventually want to support uploading to multiple buckets in the same region, so each bucket in the same region would correspond to different upload Ids - so we'd need to change this part of the code. I can update the comment.